src/app/client/src/app/modules/resource/modules/player/components/collection-player/collection-player.component.ts

```diff
@@ -600,5 +602,20 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy, AfterViewIn
       });
     });
   }
+
+  /**
+   * @since #SH-362
+   * @param  {} event
+   * @description - this method will handle the enable/disable of create course button.
+   */
+  handleSelectedItem(event) {
+    let selectedItem = 0;
+    this.collectionData['children'].forEach(item => {
```

Comment:
This will be performance issue. Everytime when any item is selected, its going through entire array to decide to enable or disable the "CreateCourse" button.

As we discussed, we just blindly relay on selectedItems array length is >0 or not. If it is greater than 0, then enable the create button.

If selected item event data is having "selected" = true, then blindly enable the button. We just parse array when event data of "selected"=false, then only we should go through array.

Reply:
done